modules/ztp-deploying-a-site.adoc

```diff
@@ -182,6 +182,6 @@ The `SiteConfig` CR creates the following CRs on the hub cluster:
 * `ClusterDeployment`
 * `InfraEnv`
 * `NMStateConfig` - One per node
-* `ExtraManifestsConfigMap` - Extra manifests. The additional manifests include workload partitioning, chronyd, mountpoint hiding, sctp enablement, and more.
+* `ExtraManifestsConfigMap` - Extra manifests. The extra manifests include workload partitioning, mountpoint hiding, sctp enablement, and more. The extra manifests are automatically merged into a single manifest per each `MachineConfigPool` role, which is named as `predefined-extra-manifests-<role>`.
```

Comment:
Hello, this will soon be an opt-in feature once [this](https://github.com/openshift-kni/cnf-features-deploy/pull/1229) PR is in. I.e Users will able to merge if and only if they are using `mergeDefaultMachineConfigs: true` in `SiteConfig`

FYI @imiller0 @tliu2021

Reply:
@pixelsoccupied Thanks Nahian! I updated it, please review. Is it only applicable from 4.11?